frontend/src/metabase/visualizations/lib/RowRenderer.js

```diff
@@ -25,16 +25,20 @@ export default function rowRenderer(
   // disable clicks
   chart.onClick = () => {};
 
-  const formatDimension = row =>
-    formatValue(row[0], { column: cols[0], type: "axis" });
+  const formatDimension = v =>
+    formatValue(v, { column: cols[0], type: "axis" });
```

Comment:
FWIW I think my reasoning for passing `row` here was that `formatDimension` references `cols[0]` anyway. 

Maybe `formatCell(row, 0)` would have been better, but not important.